pages/dmp_supporting_info/support_04_store_protect.md

```diff
@@ -62,13 +68,47 @@ It is important to keep track of the storage solutions used in the project. Usua
 How the members of your project access the data and how access is controlled can be crucial. The overall capacity and transfer speed of a storage solution can be critical parameters in case you have larger amounts of data.
 You should also consider if a more advanced way to organise your data, such as object storage or databases might be of help for your project.
 
+### Information security level(s) this storage solution will be used for
+
+While the general information security levels used at Norwegian higher educational institutions are similar (green, yellow, red, black), the classification between which data falls into which category and which solution is allowed for which security level might vary. Please consult the [institutional storage guides](support_04_store_protect#storage-guidelines) for details.
```

Comment:
Suggest to repeat links for user-friendlyness.

Reply:
```suggestion
While the general information security levels used at Norwegian higher educational institutions are similar (green, yellow, red, black), the classification between which data falls into which category and which solution is allowed for which security level might vary. Please consult the institutional storage guides for details:

* [NTNU Data storage guide](https://i.ntnu.no/wiki/-/wiki/English/Data+storage+guide)
* [UiB Storage guide](https://www.uib.no/en/foremployees/153608/storage-guide)
* [UiO Data storage guide](https://www.uio.no/english/services/it/security/lsis/storage-guide.html)
* [UiT Research data portal - Processing and storage](https://en.uit.no/research/research-dataportal/art?p_document_id=729174)
```
I think repeating the block might break the reading flow, but would leave it to you @jennyostrop